src/Core/View.js

```diff
@@ -1130,6 +1137,35 @@ class View extends THREE.EventDispatcher {
             this.notifyChange(this.camera.camera3D);
         }
     }
+
+    /**
+     * Trigger underground visualization
+     * @param {boolean} [trigger] - Enable/Disable the underground visualization.
+     */
+    setUndergroundVisualization(trigger) {
+        const atmo = this.getLayerById('atmosphere');
+        if (trigger) {
+            this.tileLayer.updateTiledLayerOpacity({ target: this });
+            this.addEventListener(VIEW_EVENTS.CAMERA_MOVED,  this.tileLayer.updateTiledLayerOpacity);
+            this.tileLayer.hideSkirt = true;
+
+
+            if (atmo) {
+                this.scene.background = new THREE.Color(0x000000);
```

Comment:
In `view_3d_map` for example, when I don't change the background we get:
![noBACK](https://github.com/iTowns/itowns/assets/126568810/94de379b-4dd5-4a3b-9edc-81d4ac45ee47)
When I don't change the atmo:
![noAtmo](https://github.com/iTowns/itowns/assets/126568810/7c98f6bd-7996-4dfa-8b55-f611483feb7d)
When I don't change anything:
![noThing](https://github.com/iTowns/itowns/assets/126568810/05101ae8-68e4-4c61-b8c5-e10d09c449d0)

With my edits:
![Screenshot from 2023-07-26 17-26-45](https://github.com/iTowns/itowns/assets/126568810/c3c8817a-f727-4a9a-80ad-8824a78d7c64)

It will be better to see the data I think.

Although, you are right about switching back to the previous value at line 1162